backpack-compose/src/main/kotlin/net/skyscanner/backpack/compose/tokens/BpkColors.kt

```diff
@@ -62,13 +62,16 @@ public class BpkColors private constructor(
   public val surfaceContrast: Color,
   public val surfaceDefault: Color,
   public val surfaceElevated: Color,
+  public val surfaceHighlight: Color,
```

Comment:
turns out these were accidentally filtered out as we were checking for ends with light